core/src/script/CGXP/widgets/FullTextSearch.js

```diff
@@ -138,17 +126,7 @@ cgxp.FullTextSearch = Ext.extend(Ext.Panel, {
         cgxp.FullTextSearch.superclass.initComponent.call(this);
 
         // define projections that may be used for coordinates recentering
-        this.projections = {};
-        if (!this.projectionCodes) {
-            this.projectionCodes = [this.map.getProjection()];
-        }
-        for (var i = 0, len = this.projectionCodes.length, code; i < len; i++) {
-            code = String(this.projectionCodes[i]).toUpperCase();
-            if (code.substr(0, 5) != "EPSG:") {
-                code = "EPSG:" + code;
-            }
-            this.projections[code] = new OpenLayers.Projection(code);
-        }
+        this.autoProjection = new OpenLayers.AutoProjection(this);
```

Comment:
Using this class requires that you add an @include statement at the beginning of the file.